src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -296,9 +296,4 @@ public boolean equals(Object other) {
                         && rooms.equals(((AddressBook) other).rooms)
                         && issues.equals(((AddressBook) other).issues));
     }
-
-    @Override
-    public int hashCode() {
```

Comment:
Hmm why is `hashCode()` being deleted?

If it is because it only accounts for residents, then it could be replaced with `Objects.hash(...)` as in UserPrefs:
![image](https://user-images.githubusercontent.com/77471078/113798532-e8604580-9785-11eb-83ec-8de0299934cb.png)